### ISSUE TITLE
Support reconstructions and similar metadata for attempts

### ISSUE THREAD

**Issue by lgarron** [cubing/cubingcontests.com#18]
For the BASC 59 Speed FMC results, we have full scrambles and reconstructions for each attempt: https://docs.google.com/spreadsheets/d/1RWEQo8jQeuZHz2mB5pDdZE2LWYA_TNdu2uUQgGlZv28/edit#gid=0

It would be nice to support a way to view this (as I would also love for the WCA to do in the future for FMC attempts).

**Comment by dmint789** [comment]
Can you provide a description of where the reconstructions would be displayed? Would they be entered into another input field during data entry?

**Comment by lgarron** [comment]
> Can you provide a description of where the reconstructions would be displayed? Would they be entered into another input field during data entry?

Yeah, each attempt would need to have an extra field for each piece of metadata (although I can make it work in a single field for Speed FMC by just using a Twizzle link).

The UI could feature this information when clicking on an attempt, or with a link icon?

**Comment by dmint789** [comment]
I think attempts being clickable is a good way to go.

What metadata needs to be stored?

**Comment by mintydev789** [comment]
Moved to [Codeberg](https://codeberg.org/mintydev/RecordRanks/issues/14)
